configs/src/deployments/json/zksyncSepolia.json

```diff
@@ -145,5 +145,162 @@
       "blockNumber": 2441217,
       "deploymentTx": "0x7bbeb9435090c339777a1f4bc62f891d2a6faa0ec331dd7a90f0800625d04a37"
     }
+  },
+  "v1.4.0": {
+    "ManagementDAOProxy": {
+      "address": "0xAEa0C1B96a4Cb95BCE15b98AFa87c32c915a3eC0",
+      "blockNumber": 2441177,
+      "deploymentTx": "0x4b7ded6dff5c99258f93733505122364acdc0878bd4b5e93d5f7878aec6d9655"
+    },
+    "ManagementDAOImplementation": {
+      "address": "0x39e836A6c32163733929B213965e3feC0007914a",
+      "blockNumber": 4892833,
+      "deploymentTx": "0x39c61bf46f0237c0be786e10254fce52615f95419fbb885600c6529e5b34ccdc"
+    },
+    "DAOFactory": {
+      "address": "0xee321f16f7F0a0F0d8b850E70c4eAde4A288ECd7",
+      "blockNumber": 4892833,
+      "deploymentTx": "0x39c61bf46f0237c0be786e10254fce52615f95419fbb885600c6529e5b34ccdc"
+    },
+    "DAOBase": {
+      "address": "0x39e836A6c32163733929B213965e3feC0007914a",
+      "blockNumber": 4892833,
+      "deploymentTx": "0x39c61bf46f0237c0be786e10254fce52615f95419fbb885600c6529e5b34ccdc"
+    },
+    "DAORegistryProxy": {
+      "address": "0x9305Ecd8D3837F7AC4f425dA578a0D11352D317A",
+      "blockNumber": 2441205,
+      "deploymentTx": "0x8afe5b11a6ff00b478a507ed9acc60a47e7d9b5c8dc43867da4571e87c947186"
+    },
+    "DAORegistryImplementation": {
+      "address": "0x044058e7385D3f0EC0e7Cc759192A28007549bF4",
+      "blockNumber": 4892835,
+      "deploymentTx": "0x8037392beaf6536572e15ce35acda014be39114efc6a0e82784d53fce88019a9"
+    },
+    "DAOENSSubdomainRegistrarProxy": {
+      "address": "0x753A6c93189553d5964C121900A76003c5560bC0",
+      "blockNumber": 2441197,
+      "deploymentTx": "0x78dba520e9a00ad2f336c13213eff71673e936b2a5c938723671788930dda241"
+    },
+    "DAOENSSubdomainRegistrarImplementation": {
+      "address": "0xbd29608a966beDBfA1E82D3Fe42a33c45E546d94",
+      "blockNumber": 2441196,
+      "deploymentTx": "0x9ae4ec0e969268471bf390833cb1fc40c77bf9bfbbcb06288d20b9197c63c949"
+    },
+    "ENSRegistry": {
+      "address": "0xD9d4642F2C04f25c2b4373f5D67D4358e72Ab3B2",
+      "blockNumber": 2441182,
+      "deploymentTx": "0x728c28445f45f0e9c534d9805ca4f83af6f113cca7ed316e6b60ce373482dc15"
+    },
+    "PlaceholderSetup": {
+      "address": "0xb0E4DaEbe52E65F0baFb53342DaCBB89bF5c6e9b",
+      "blockNumber": 2441231,
+      "deploymentTx": "0x153b3e8fd75477e99d30e39d355c52abb598c992ee64423854722904b19d27fe"
+    },
+    "PluginRepoBase": {
+      "address": "0xb9F2E71f82F1ab801136Fdd60176D0aCF298735E",
```

Comment:
Source not verified

Reply:
https://github.com/aragon/osx-commons/pull/146#discussion_r2012213626